pages.de/common/cp.md

```diff
@@ -22,3 +22,7 @@
 - Kopiere alle Textdateien in einem Verzeichnis und warte auf eine Bestätigung, falls eine Datei überschrieben werden soll:
 
 `cp -i {{/pfad/zu/*.txt}} {{pfad/zu/ziel_verzeichnis}}`
+
+- Dereferenziere symbolische Links vor dem Kopieren:
+
+`cp -L {{link}} {{pfad/zu/ziel_verzeichnis}}`
```

Comment:
```suggestion
`cp -L {{link}} {{pfad/zu/zielverzeichnis}}`
```
```suggestion
`cp -L {{link}} {{pfad/zu/ziel-verzeichnis}}`
```

Either works.

Reply:
We can't use the second one, because we don't usually use `-` in agruments. But wouldn't you think it would be better giving up a little bit of grammatical correctness for readability and keeping `ziel_verzeichnis`?

Reply:
Well… It just sounds a bit… Idk, off, weird. IMHO, `zielverzeichnis` is readable enough, but I have no idea what others think…

Reply:
Any German word probably looks completely unreadable to a non-german 😄.
But I'm fine we `zielverzeichnis`.

Reply:
It's not readable to me at all, but neither is `ziel_verzeichnis`, for that matter ;)